ReText/window.py

```diff
@@ -1105,6 +1109,9 @@ def insertFormatting(self, formatting):
 			dialog.show()
 			self.formattingBox.setCurrentIndex(0)
 			return
+		elif formatting == 'image':
+			self.insertImage()
+			return
```

Comment:
And what about it. This makes the `Formatting -> image` doesn't insert the `![Alt text](*URL*)`, but opens the select image box instead. Do you thing is better remove it to insert the `![Alt text](*URL*)`, or keeps it to open the select image box?

Reply:
Let's have different actions for people who want different things:

- __Formatting → image__ — to insert any image (e.g. remote one).
- __Edit → Insert images by file path__ — to insert image from local file.

I will fix it myself.